src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -3,38 +3,27 @@
     "dnxcore50": {
       "dependencies": {
         "NuProj": "0.10.4-beta-gf7fc34e7d8",
+        "MSBuild": "0.1.0-preview-00022",
         "Microsoft.Build.Framework": "0.1.0-preview-00022",
         "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
         "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
         "Microsoft.Build.Targets": "0.1.0-preview-00022",
         "Microsoft.Build": "0.1.0-preview-00022",
+        "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160429-01",
+        "Microsoft.CodeAnalysis.VisualBasic": "1.3.0-beta1-20160429-01",
```

Comment:
I had to add this two because of a dependency conflict between Microsoft.NETCore.App and Microsoft.Net.Compilers.NetCore.